packages/ckeditor5-bookmark/src/bookmarkui.ts

```diff
@@ -511,6 +522,8 @@ export default class BookmarkUI extends Plugin {
 		let prevSelectedBookmark = this._getSelectedBookmarkElement();
 		let prevSelectionParent = getSelectionParent();
 
+		this._updateFormButtonLabel();
```

Comment:
Instead of calling this twice in `_startUpdatingUI()`, can we just update the button text once at the end of the `_showUI()`?

Reply:
I thought about it but found an edge case. In that case if balloon is not closed when you back to the form view the button is not updated, so we need to initialize properly the value of the button and updated properly.

[screen-capture (1).webm](https://github.com/user-attachments/assets/d53ac37b-e9c7-4d6c-9fe6-6c260731cbef)

Reply:
Do we have test for this edge case?

Reply:
No, but right on my way to add it